internal/tfengine/tfengine.go

```diff
@@ -168,3 +174,21 @@ func ConvertToLocalBackend(path string) error {
 
 	return nil
 }
+
+func formatDir(dir string) error {
```

Comment:
move this to a common location? I can use it to format org policies as well in policygen